modules/transactionpool/subscribe_test.go

```diff
@@ -0,0 +1,78 @@
+package transactionpool
+
+import (
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+	"testing"
+)
+
+// mockSubscriber receives transactions from the transaction pool it is
+// subscribed to, retaining them in the order they were received.
+type mockSubscriber struct {
+	txns []types.Transaction
+}
+
+// newMockSubscriber returns a blank mockSubscriber that is ready to subscribe
+// to a transaction pool.
+func newMockSubscriber() mockSubscriber {
+	return mockSubscriber{}
+}
```

Comment:
constructors should only be used when some initialization needs to be performed.